src/html/parse-markdown.js

```diff
@@ -23,7 +24,12 @@ function parseMarkdown(context, action) {
   const { extension } = request;
 
   action.logger.debug(`Parsing markdown from request body starting with ${body.split('\n')[0]}`);
-  content.mdast = unified().use(remark).parse(body);
+  content.mdast = unified()
+    .use(remark, {
+      position: false,
```

Comment:
👍 That essentially replaces my other PR. Now if we want to keep positions in there in case of debug logs, we may need to make this configurable. I'd propose to not address the position removal here and keep it in the other PR for clarity